environment/src/main/java/com/sequenceiq/environment/environment/flow/DatalakeMultipleFlowsResultEvaluator.java

```diff
@@ -0,0 +1,83 @@
+package com.sequenceiq.environment.environment.flow;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
Please remove. We usually do not use JetBrains specific annotations like this. (I know IDEA adds this automatically for certain refactors.)

Reply:
ok